index.md

```diff
@@ -106,28 +106,28 @@ extra_footer: |
           Struct language (<code>.ksy</code> file) and then can be compiled with
           <code>ksc</code> into source files in one of the supported programming
           languages. These modules will include a generated code for a parser
-          that can read described data structure from a file / stream and give
+          that can read the described data structure from a file or stream and give
           access to it in a nice, easy-to-comprehend API.
         </p>
         <h2 id="using-ks-in-your-project">Using KS in your project</h2>
-        <p>Typically, using formats described in KS in your project, involves the
+        <p>Typically, using formats described in KS in your project involves the
           following steps:
         </p>
         <ul>
           <li>Describe the format — i.e. create a <code>.ksy</code> file</li>
-          <li>Use visualizer to debug the format and ensure that it parses data properly</li>
-          <li>Compile <code>.ksy</code> file into target language source file and include
+          <li>Use the visualizer to debug the format and ensure that it parses data properly</li>
```

Comment:
I would prefer

```suggestion
          <li>Use a visualizer to debug the format and ensure that it parses data properly</li>
```

because there are more of them: [`kaitai_struct_visualizer`](https://github.com/kaitai-io/kaitai_struct_visualizer), [`kaitai_struct_webide`](https://github.com/kaitai-io/kaitai_struct_webide), [`kaitai_struct_gui`](https://github.com/kaitai-io/kaitai_struct_gui) and a few other external projects ([Kiewtai](https://github.com/taviso/kiewtai), [Hobbits](https://www.youtube.com/watch?v=RIACHS5HEuE&feature=youtu.be&ab_channel=MahletInc) etc.).